Content.Server/StationRecords/Systems/RecordsConsoleSystem.cs

```diff
@@ -0,0 +1,41 @@
+using Content.Shared.StationRecords;
+
+namespace Content.Server.StationRecords.Systems;
+
+/// <summary>
+/// Currently just a helper for all records consoles, but could handle some parts of ui in the future.
+/// </summary>
+public sealed class RecordsConsoleSystem : EntitySystem
```

Comment:
GeneralStationRecordSystem is already the system made for generalizing consoles so i dont see the point in adding another one with the same purpose, prob just move these methods there

Reply:
isnt that the backend for station records console specifically

Reply:
Got the name wrong, this naming scheme is bullshit, meant `StationRecordsSystem` (still more tied to station records than anything but it's less confusing to put general stuff in there for now instead of a dummy system with very little usage)

Reply:
also adding another thing to it